node-fs-writefile/write-rand.js

```diff
@@ -0,0 +1,17 @@
+import { writeFile } from 'node:fs/promises';
+
+function randomNum(max) {
```

Comment:
This function produces a random integer between 0 and max - 1. According to the instructions, any random number, even a non-integer, would be acceptable. But, it's fun code!